app/src/server/utils/convertFunctionCalls.ts

```diff
@@ -31,7 +41,29 @@ export const convertFunctions = (
   }));
 };
 
-export const convertMessages = (
+export const convertToolsToFunctions = (
+  tools: ChatCompletionCreateParamsBase["tools"],
+): ChatCompletionCreateParamsBase["functions"] => {
+  if (!tools) return tools;
+  return tools.map((tool) => ({
```

Comment:
I'm probably missing something but just looking at the logic couldn't this be written as `return tools?.map((tool) => tool.function)`?

Reply:
No, I think you're right. I had it structured this way because of some legacy logic that I've since removed, so I'll update it to be sleeker.